_config.yml

```diff
@@ -220,26 +212,9 @@ apple-touch-icon-114x114-precomposed:   "apple-touch-icon-114x114.png"
 apple-touch-icon-76x76-precomposed:     "apple-touch-icon-76x76.png"
 apple-touch-icon-72x72-precomposed:     "apple-touch-icon-72x72.png"
 apple-touch-icon-precomposed:           "apple-touch-icon.png"
-msapplication_tileimage:                "mstile-150x150.png"               # Size 144x144
+msapplication_tileimage:                "mstile-150x150.png"  # Size 144x144
 msapplication_tilecolor:                "#ffffff"
 
-
-#       ______                __                __
```

Comment:
But the contextual knowledge will very likely be lost and someone will have to dig hard to find out how to reintroduce this (rather than do a simple grep in the repo). Whatever our personal preferences on FB, removing it, to me, hurts more than it helps. At the end of the day it is a potentially useful comment.

It is pretty common to have config files that retain all the configuration options commented out. When there is no easy way to list all available options, this is a pretty reasonable approach.

Reply:
Somewhat agreed in principal, but clicking through the shortened link reveals that the Facebook OpenGraph interface is deprecated. This would have to be done over again anyway, so I suggest keeping it out. I will, however, make a note in `_config.yml` that such an integration may be possible.

Reply:
Partially restored by e74a34f.

Reply:
Resolved by 295ab288.